src/plugins/store.js

```diff
@@ -23,6 +23,7 @@ export const store = new Vuex.Store({
 		populatedInterfaceList: [],
 		noSteamVR: false,
 		currentLibraryFolder: "No library folder selected.",
+        currentError: ""
```

Comment:
Indenting suspect.

Reply:
That indenting was correct, it was all the surrounding code that was incorrect! It's been fixed. :)